app/views/games/show.html.erb

```diff
@@ -0,0 +1,5 @@
+<h1>Chessboard</h1>
+
+<table id='chessboard'>
```

Comment:
If you'd like, you can actually construct the board with erb (ruby) rather than JavaScript.  We _might_ find that easier later to place pieces on the board (or if the JS approach makes more sense to you, go for it, mainly just pointing out a possibility):

```erb
<table id="chessboard">
<% (0..7).each do |y| %>
  <tr>
  <% (0..7).each do |x| %>
    <td data-x="<%= x %>" data-y="<%= y %>"></td>
  <% end %>
<% end %>
</table>
```